src/server/handlers/github.js

```diff
@@ -91,11 +91,11 @@ export const getSnapcraftData = (repositoryUrl, token) => {
   const { owner, name } = parseGitHubRepoUrl(repositoryUrl);
   const cacheId = getSnapNameCacheId(repositoryUrl);
 
-  return new Promise((resolve) => {
-    getMemcached().get(cacheId, (err, result) => {
-
-      if (!err && result !== undefined) {
-        return resolve(result);
+  return getMemcached().get(cacheId)
+    .catch(() => undefined)
```

Comment:
This looks quite strange to silently ignore error and return `undefined`.
It at least asks for a comment in code to explain why it is like that or log error.

And as far as I can see you use such pattern later in launchpad handler:

```
.catch((err) => {
  logger.error(`Error deleting ${cacheId} from memcached:`, err);		            
})
```

This would handle the error at least by logging it and also (silently) return undefined.